test/schema_compatibility/test_schema_compatibility.rb

```diff
@@ -397,6 +403,14 @@ def a_dint_b_dint_record1_schema
     Avro::Schema.parse('{"type":"record", "name":"Record1", "fields":[{"name":"a", "type":"int", "default":0}, {"name":"b", "type":"int", "default":0}]}')
   end
 
+  def nested_record
+    Avro::Schema.parse('{"type":"record","name":"parent","fields":[{"name":"attribute","type":{"type":"record","name":"child","fields":[{"name":"id","type":"string"}]}}]}')
+  end
+
+  def nested_nullable_record
```

Comment:
Would `nested_optional_record` be a better description here?

Reply:
changed